glue/core/data_factories/gridded.py

```diff
@@ -71,10 +71,8 @@ def is_gridded_data(filename, **kwargs):
 
 gridded_data.label = "FITS/HDF5 Image"
 gridded_data.identifier = is_gridded_data
+gridded_data.priority = 2
```

Comment:
Yes, it was for backward-compatibility in case anyone was defining it like us here. But then it's probably true that most other people are using the decorator (in which case ``priority`` does have a default value). Maybe we should in fact also use the decorator internally, then ``priority`` would always be defined.